docs/notebooks/tutorials/QFM_surface.ipynb

```diff
@@ -0,0 +1,783 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    )
</pre><p>Can you put semicolon to the end, I get triggered by the text of matplotlib :D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/1264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>